drivers/telescope/lx200_10micron.cpp

```diff
@@ -60,7 +58,7 @@ LX200_10MICRON::LX200_10MICRON() : LX200Generic()
 
     SetTelescopeCapability( TELESCOPE_CAN_GOTO | TELESCOPE_CAN_SYNC | TELESCOPE_CAN_PARK | TELESCOPE_CAN_ABORT |
         TELESCOPE_HAS_TIME | TELESCOPE_HAS_LOCATION | TELESCOPE_HAS_PIER_SIDE |
-        TELESCOPE_HAS_TRACK_MODE | TELESCOPE_CAN_CONTROL_TRACK | TELESCOPE_HAS_TRACK_RATE );
+        TELESCOPE_HAS_TRACK_MODE | TELESCOPE_CAN_CONTROL_TRACK | TELESCOPE_HAS_TRACK_RATE | TELESCOPE_CAN_TRACK_SATELLITE );
```

Comment:
I'll move these flags to one per line later.

Reply:
I went ahead and did it :)

Reply:
thanks :)